<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>manualRotator</class>
    <widget class="QDialog" name="manualRotator">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Manual Rotator</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_1">
            <property name="spacing">
                <number>3</number>
            </property>
            <property name="leftMargin">
                <number>3</number>
            </property>
            <property name="topMargin">
                <number>3</number>
            </property>
            <property name="rightMargin">
                <number>3</number>
            </property>
            <property name="bottomMargin">
                <number>3</number>
            </property>
            <item>
                <widget class="QLabel" name="help">
                    <property name="text">
                        <string>&lt;p&gt;To disable rotation validation, uncheck the Rotator option in Alignment Options.&lt;/p&gt;</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="spacing">
                        <number>3</number>
                    </property>
                    <property name="topMargin">
                        <number>10</number>
                    </property>
                    <item>
                        <layout class="QGridLayout" name="gridLayout">
                            <item row="0" column="0" colspan="1">
                                <widget class="QLabel" name="currentRotationL">
                                    <property name="text">
                                        <string>Current Rotation</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0" colspan="1">
                                <widget class="QLineEdit" name="currentRotation">
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1" colspan="1">
                                <widget class="QLabel" name="targetRotationL">
                                    <property name="text">
                                        <string>Target Rotation</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1" colspan="1">
                                <widget class="QLineEdit" name="targetRotation">
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLabel" name="icon">
                    <property name="alignment">
                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                    </property>
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>0</y>
                            <width>350</width>
                            <height>350</height>
                        </rect>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="diff">
                    <property name="alignment">
                        <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>64</pointsize>
                            <bold>true</bold>
                        </font>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_1">
                    <property name="spacing">
                        <number>3</number>
                    </property>
                    <item>
                        <widget class="QPushButton" name="takeImageB">
                            <property name="text">
                                <string>Take Another Image</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="cancelB">
                            <property name="text">
                                <string>Cancel Load &amp;&amp; Slew</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
